docs/libindy/macos.md

```diff
@@ -4,6 +4,16 @@
 
 To see if Libindy is already installed, execute the following command:
 
+> A better test would be to clone the [indy-sdk](https://github.com/hyperledger/indy-sdk/) and run:
+>
+> ```
+> cd indy-cli
+> cargo build --release
+> cd ./target/release/
+> ./indy-cli
+> wallet create 123 key
+> ```
```

Comment:
Should we just replace the old method then?

Reply:
This method would require cloning the indy-sdk repository and building via Cargo. The other check might be a bit less accurate, but it only requires the `ls` command. 

The issue with this test is that libindy can be installed correctly for the indy cli, but not for node. 

I will see if there is a better way, e.g. clone a small repo that just runs `indy.createWallet({id:'12'},{key:'abc'});` to test.

Reply:
A thought. What if we add a binary to the `@aries-framework/node` package. This way you can just use `npx` to check if the installation was succeful

With [this PR](https://github.com/hyperledger/aries-framework-javascript/pull/510) merged we will be able to (@blu3beri I copied your example from https://github.com/animo/test-libindy-installation-for-node)

```
npx -p @aries-framework/node is-indy-installed

# output
# Libindy was installed correctly
```

What do you think?

Reply:
That's awesome! Way better than what I had in mind.